doc/glossary/index.rst

```diff
@@ -31,3 +31,374 @@ Glossary
    Counter
         The token property ``count`` in privacyIDEA is used to calculate the :term:`OTP Value`
         using the HMAC-type algorithms HOTP or TOTP.
+
+   WebUI
+        privacyIDEA comes with a web-based user interface which is used to manage and
+        configure the privacyIDEA server.
+
+        It is also used a self-service portal for the average user, who manages his own tokens.
+        This section gives an overview on the interface and links the respective sections in the documentation.
+
+        * :ref:`dashboard`
+        * :ref:`tokensview`
+        * :ref:`usersview`
+        * :ref:`machines`
+        * :ref:`config`
+        * :ref:`audit`
+        * :ref:`components`
+
+   Time Step
+        A TOTP token can have a timestep of 30/60 seconds. It can still be used, if this 30/60 seconds are over.
+        Technically the timestep is the divider by which the seconds since 1.1.1970 (unix system time)
+        are divided to calculate the OTP value.
+
+        The ``timestep`` is different to the :ref:`Time Window`.
```

Comment:
I think you do not have defined a reference anchor for "Time Window".

Reply:
done in deb552c3c5b4a7656793fc1d800c15c2286195d4.